src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/EnvelopeProcessResult.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.processedenvelopes;
+
+public class EnvelopeProcessResult {
```

Comment:
I think it should be 'processing'

Reply:
but it is end product, it should finished, completed successfully

Reply:
Theoretically, processing is correct, but given we've got `ProcessResult` already for callbacks, makes sense to keep names consistent.

Reply:
I am making it Processing

Reply:
when it is Processing I expect the class to cover all results, exceptions everything and return to me as result.but we just return this class if it is scuccessfull.